src/dialogue.h

```diff
@@ -256,11 +248,10 @@ struct dialogue {
         std::vector<mission *> missions_assigned;
 
         talk_topic opt( dialogue_window &d_win, const std::string &npc_name, const talk_topic &topic );
-
         dialogue() = default;
-        talker *actor( const bool is_beta ) const {
-            return ( is_beta ? beta : alpha ).get();
-        }
+        explicit dialogue( std::unique_ptr<talker> alpha_in,
```

Comment:
This doesn't need to be `explicit`, as it has two arguments, but it is not a problem if it is.
(`explicit` means that when an object of this type is constructed, it needs to explicitly call a constructor).